tests/jvm/src/test/resources/metac.expect

```diff
@@ -1801,6 +1801,8 @@ Occurrences:
 [24:20..24:21): x <= example/InstrumentTyper#singletonType().(x)
 [24:38..24:41): ??? => scala/Predef.`???`().
 [25:12..25:19): clazzOf <= example/InstrumentTyper#clazzOf.
+[25:22..25:29): classOf => scala/Predef.classOf().
+[25:30..25:41): Option[Int] => scala/Option#
```

Comment:
This means that renaming `Option` to `Foobar` would replace `Option[Int]` with `Foobar`

Reply:
Fixed. You might want to check out this solution as it's based on character occurrence in string - I'm not sure if it's a proper way to do it.